src/test_l1_tvSettings.c

```diff
@@ -14540,7 +14821,7 @@ void test_l1_tvSettings_negative_GetPQParams (void)
 	if (extendedEnumsSupported == true)
 	{
 		/* Step 13: Calling tvsettings GetPQParams after TvTermm and expecting the API to return tvERROR_INVALID_STATE */
-		result = GetPQParams(Configfile.picmodeStruct.pqValue[0], (tvVideoSrcType_t)Configfile.videoSrcStruct.videoSourceValue[0], (tvVideoFormatType_t)Configfile.videoFormtStruct.videoFormatValue[0], (tvPQParameterIndex_t)Configfile.pq_paramIndex.videoSourceValue[0], &value);
+		result = GetPQParams(pqValue, (tvVideoSrcType_t)videoSource, (tvVideoFormatType_t)videoFormat, (tvPQParameterIndex_t)Configfile.pq_paramIndex.videoSourceValue[0], &value);
```

Comment:
Configfile.pq_paramIndex.videoSourceValue is still present through out the function.

Reply:
Replaced